src/Node/FS/Constants.purs

```diff
@@ -34,7 +37,7 @@ foreign import copyFile_FICLONE :: CopyMode
 -- |  	If present, the copy operation will attempt to create a copy-on-write reflink. If the underlying platform does not support copy-on-write, then the operation will fail with an error.
 foreign import copyFile_FICLONE_FORCE :: CopyMode
 
-defaultCopyMode = copyFile_EXCL :: CopyMode
+defaultCopyMode = copyFile_NO_FLAGS :: CopyMode
```

Comment:
I don't think this is a good default, overwriting should be opt-in since it's more dangerous.

```suggestion
defaultCopyMode = copyFile_EXCL :: CopyMode
```

Reply:
but it is the default option by nodejs documentation

https://github.com/purescript-node/purescript-node-fs/issues/81#issuecomment-1752230419